src/corelibs/U2View/src/ov_msa/CreateSubalignmentDialogController.cpp

```diff
@@ -200,7 +201,7 @@ void CreateSubalignmentDialogController::accept() {
         QMessageBox::critical(this, this->windowTitle(), tr("No path specified"));
         return;
     }
-    if (fi.baseName().isEmpty()) {
+    if (fi.baseName().isEmpty() || possibleDir.exists()) {
```

Comment:
You can use https://doc.qt.io/qt-5/qfileinfo.html#isDir to check if file-info is dir and do not create a new variable here.